cli/tools/upgrade.rs

```diff
@@ -449,16 +453,18 @@ pub async fn upgrade(
       }
     }
     log::info!("Upgraded successfully");
-    log::info!(
-      "{}{}",
-      "Release notes: https://github.com/denoland/deno/releases/tag/v",
-      &install_version,
-    );
-    log::info!(
-      "{}{}",
-      "Blog post: https://deno.com/blog/v",
-      get_minor_version(&install_version)
-    );
+    if !upgrade_flags.canary {
+      log::info!(
+        "{}{}",
+        "Release notes: https://github.com/denoland/deno/releases/tag/v",
+        &install_version,
+      );
+      log::info!(
+        "{}{}",
+        "Blog post: https://deno.com/blog/v",
+        get_minor_version(&install_version)
+      );
+    }
```

Comment:
Same code three times. Maybe we should extract this out.

Reply:
I was thinking as well (there actually is 2 more places), but they dont all use log::info, which is why i held off with that

Reply:
It looks like we can just use log::info everywhere (side note: IMO it would be nice to do the `!has_same_minor_version` check to conditionally show the blog post url everywhere too)

Reply:
ok, done